app/renderer/actions/loginForm.ts

```diff
@@ -1,11 +1,19 @@
 import { actionCreator } from "./helpers"
 import * as Actions from "./actionNames"
+import { ComputedTransaction } from "app/renderer/prefilledTransactions"
+
 import { Wallet } from "app/common/runtimeTypes/storage/wallets"
 import {
+  SaveTransactionsAction,
   SaveWalletAction,
 } from "./actionTypes"
 
 // Action creator to save wallet
 export const saveWallet = (wallet: Wallet): SaveWalletAction => {
   return actionCreator<Wallet>(Actions.SAVE_WALLET)(wallet)
+}
+
+// Action creator to save wallet
+export const saveTransactions = (transactions: ComputedTransaction): SaveTransactionsAction => {
+  return actionCreator<ComputedTransaction>(Actions.SAVE_TRANSACTIONS)(transactions)
```

Comment:
Same thing here, it is not a single transaction that needs to be stored in redux, so the parameter should be of type `ComputedTransactions`. Also, the comment needs to be updated.